client/overview/style.scss

```diff
@@ -1 +1,34 @@
 @import 'node_modules/@woocommerce/experimental/src/style.scss';
+
+// Adapted from woocommerce-admin:client/homescreen/style.scss
+.woocommerce-payments-page.overview {
+	.components-card__header.is-size-large,
```

Comment:
This change applies to card elements on overview page beyond the Task List. It actually breaks styling in #1464. I am updating it to only affect `wooocommerce-task-card` class for now, but it's better to change it separately and update in the `develop` branch.

Reply:
Thanks for that fix @dmvrtx! If #1464 isn't going to get merged soon I can open a separate PR against `develop`.